config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -53,9 +53,18 @@ config SYSTEM_WORKQUEUE_STACK_SIZE
     default 2560 if CHIP_WIFI
     default 2048 if CHIP_ICD_LIT_SUPPORT
 
+if CHIP_WIFI
 config HEAP_MEM_POOL_SIZE
-    default 110720 if CHIP_WIFI
+    default 48336 if BUILD_WITH_TFM
+    default 40144
+
+config NRF_WIFI_DATA_HEAP_SIZE
+    default 50576 if BUILD_WITH_TFM
+    default 42384
 
+config NRF_WIFI_CTRL_HEAP_SIZE
+    default 20000
+endif # CHIP_WIFI
```

Comment:
better to leave this default, no? It shouldn't be controlled by samples?

Reply:
Agreed. Done.